src/migrations/migrations/migrateNotificationSettingsToV3.ts

```diff
@@ -0,0 +1,21 @@
+import { Migration, MigrationName } from '@/migrations/types';
+import { getAllWalletNotificationSettingsFromStorage, setAllWalletNotificationSettingsToStorage } from '@/notifications/settings/storage';
+
+export function migrateNotificationSettingsToV3(): Migration {
+  return {
+    name: MigrationName.migrateNotificationSettingsToVersion3,
+    async migrate() {
+      const walletSettings = getAllWalletNotificationSettingsFromStorage();
+
+      // reset successfullyFinishedInitialSubscription
+      if (walletSettings.length) {
```

Comment:
Reason I ask is because `JSON.parse()` and casting it gets us the desired type, but wondering if there could be any instances of it not being the correct type.

Reply:
will always be `[]`